crates/nargo_cli/tests/test_data_ssa_refactor/debug_logs/src/main.nr

```diff
@@ -19,6 +18,8 @@ fn main(x : Field, y : pub Field) {
     let s = myStruct { y: x, x: y };
     std::println(s);
 
+    std::println(f"randomstring{x}{x}");
```

Comment:
This was fixed by only adding an underscore to the regex?

Reply:
No I had an incorrect regex in the resolver that did not match the one inside of `foreign_calls.rs`. I changed `r"\{([\S]+)\}"` to `r"\{([a-zA-Z0-9_]+)\}"`